src/config/dynamic-theme-fixes.config

```diff
@@ -5468,6 +5468,18 @@ img[src^="data:image/png;"]
 
 ================================
 
+play.google.com/books/listen
+
+INVERT
+.gb_ua
```

Comment:
Change it to `a[href*="about/products"]` this selector is known to change every x weeks.

Reply:
`a[href*="about/products"]` doesn't seem to work in this case. I used `a[href*="books/audiobooks"]` instead.